tools/build_image.py

```diff
@@ -18,7 +18,7 @@
 
 print('Creating AWS EC2 instance')
 instances = ec2.create_instances(
-    ImageId=config.DEBIAN_BASE_IMAGE_ID,
+    ImageId=config.UBUNTU_BASE_IMAGE_ID,
```

Comment:
Why are we not using debian anymore? Can it break something with the [migration script](https://github.com/meilisearch/meilisearch-digitalocean/pull/286/files)?

Reply:
Because of this [issue](https://github.com/meilisearch/meilisearch/issues/2850#event-7534950058) Debian-10 doesn't have the good `glibc` version we could stay on Debian-11 but I choose to pass on Ubuntu20 instead.